utils/src/treev2/quickPickWizard/ContextValueQuickPickStep.ts

```diff
@@ -0,0 +1,89 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as types from '../../../index';
+import { GenericQuickPickOptions, GenericQuickPickStep } from './GenericQuickPickStep';
+import { isAzExtParentTreeItem } from '../../tree/InternalInterfaces';
+import { QuickPickWizardContext } from './QuickPickWizardContext';
+
+/**
+ * Describes filtering based on context value. Items that pass the filter will
+ * match at least one of the `include` filters, but none of the `exclude` filters.
+ */
+export interface ContextValueFilter {
+    /**
+     * This filter will include items that match *any* of the values in the array.
+     * When a string is used, exact value comparison is done.
+     */
+    include: string | RegExp | (string | RegExp)[];

```

Comment:
Should we allow strings at all, or just `RegExp`? I'm worried the behavior could get unintuitive / confusing quickly when you consider that context values are now being treated as an array (sometimes), and are being sourced from multiple extensions.

Reply:
Duplicated here: https://github.com/microsoft/vscode-azuretools/pull/1229#discussion_r975785370